megamek/src/megamek/common/AmmoType.java

```diff
@@ -24,6 +25,8 @@
 
 public class AmmoType extends EquipmentType {
 
+    private static final MMLogger LOGGER = MMLogger.create(AmmoType.class);
```

Comment:
nitpick, the standard for logger in the project is logger lowecase.

Reply:
I noticed :) 
The very widely adopted convention for constants (static final) however is ALL_CAPS. When Tex reworked the logger thing, he made another million changes and I didnt want to pick on that minuscule issue. But I'd really like to stay with our style guide and common styles unless there's good reason. Is there one?

Reply:
@rjhancock This kinda involves you :)

Reply:
This is a case of when I did the mass changes I was less worried about style guide and more worried about ensuring legal compliance.

Style guide says upper case (which is a best practice I ignored in this case due to a 2 week marathon run of more important issues), then it should be upper case.